gc_ruboconfig.gemspec

```diff
@@ -11,7 +11,10 @@ Gem::Specification.new do |spec|
 
   spec.files         = ['rubocop.yml', 'rails.yml']
   spec.add_dependency 'rubocop', '>= 1.63'
+  spec.add_dependency 'rubocop-capybara', '>= 2.21.0'
+  spec.add_dependency 'rubocop-factory_bot', '>= 2.26.1'
   spec.add_dependency 'rubocop-performance', '>= 1.21'
   spec.add_dependency 'rubocop-rails', '>= 2.25.0'
-  spec.add_dependency 'rubocop-rspec', '>= 2.29.2'
+  spec.add_dependency 'rubocop-rspec', '>= 3.0.1'
+  spec.add_dependency 'rubocop-rspec_rails', '>= 2.30.0'
```

Comment:
Can we make these optional dependencies for repos that don't use them?

Reply:
Seperated out capybara